pkg/app/context.go

```diff
@@ -819,7 +819,9 @@ func (ctx *RequestContext) Copy() *RequestContext {
 func (ctx *RequestContext) Next(c context.Context) {
 	ctx.index++
 	for ctx.index < int8(len(ctx.handlers)) {
+		tempParams := ctx.Params
```

Comment:
It's not very elegant to handle it here. Besides every time you call this function, this logic will be executed. 
You can see from the whole process and design ideas of Hertz which stage is more appropriate for processing

Reply:
Just pushed a commit that revert this change, and provide another solution, please check it out.